src/internal/kopia/upload.go

```diff
@@ -884,6 +885,17 @@ func inflateDirTree(
 		return nil, errors.Wrap(err, "inflating collection tree")
 	}
 
+	baseIDs := make([]manifest.ID, 0, len(baseSnaps))
+	for _, snap := range baseSnaps {
+		baseIDs = append(baseIDs, snap.ID)
+	}
+
+	logger.Ctx(ctx).Infow(
+		"merging hierarchies from base snapshots",
+		"snapshot IDs",
```

Comment:
Try to avoid spaces in logging keys, please.  It'll make the json much easier to interact with down the line.  I don't know if we've settled on snake or camel case, but either should work here.